ethpm_cli/parser.py

```diff
@@ -165,32 +170,71 @@ def auth_action(args: argparse.Namespace) -> None:
     Config(args)
     try:
         authorized_address = get_authorized_address()
-        cli_logger.info(f"Keyfile stored for address: 0x{authorized_address}.")
+        cli_logger.info(f"Keyfile stored for address: {authorized_address}.")
     except AuthorizationError:
         cli_logger.info(
-            "No valid keyfile found. Use `ethpm auth --keyfile-path <path_to_keyfile>` "
-            "to set your keyfile for use with ethPM CLI."
+            "No valid keyfile found. "
+            "If you already have a keyfile, use `ethpm auth link --keyfile-path [path_to_keyfile]` "
+            "to set your keyfile for use with ethPM CLI. \n"
+            "To generate an encrypted keyfile, use `ethpm auth init`"
         )
 
 
+def auth_link_action(args: argparse.Namespace) -> None:
+    Config(args)
+    if "keyfile_path" not in args or not args.keyfile_path:
+        cli_logger.info("Invalid --keyfile-path flag")
+    else:
+        link_keyfile(args.keyfile_path)
+
+
+def auth_unlink_action(args: argparse.Namespace) -> None:
+    Config(args)
+    unlink_keyfile()
+
+
+def auth_init_action(args: argparse.Namespace) -> None:
+    Config(args)
```

Comment:
This is a code smell to me since I'm assuming you are relying on some sort of side-effect action that happens by instantiating this object?  If so I would advocate making this API more explicit so that it is clear what is happening here and why.  At minimum a code comment, but I think that the side effect nature of object instantiation isn't ideal and should potentially be moved into something like a `classmethod` which handles the side-effect stuff and keep the validation checks in the `__init__`...